modules/@apostrophecms/area/ui/apos/apps/AposAreas.js

```diff
@@ -118,31 +118,16 @@ export default function() {
   }
 
   function createWidgetClipboardApp() {
-    class Clipboard {
```

Comment:
I simplified this class a lot but didn't change the interface. While I can't see any bug in the original logic for it, the "storage" event just wasn't firing for me when copying to the clipboard in a separate tab. This simplified logic always works because it always talks to the clipboard.